modules/core/src/main/java/org/apache/synapse/mediators/builtin/CallMediator.java

```diff
@@ -341,7 +341,7 @@ public void destroy() {
 
     @Override
     public boolean isContentAware() {
-        return false;
+        return blocking;
```

Comment:
Shall we add a comment explaining the change?

Reply:
Done